src/pages/fresh_install/FreshInstallPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import '../fresh_install/FreshInstallPage.css';
+import path4 from "../../images/path4.svg";
+import classNames from 'classnames';
+
+export default class FreshInstallPage extends React.Component<{}, { selectedTemperatureScale: string }> {
```

Comment:
For the code readability, I'd suggest us to extract the type definitions in `type` constants, like:

```js
type Props = { ... };
type State = { ... };

class FreshInstallPage extends React.Component<Props, State> {
```

Reply:
Great, thanks for the suggestion. Done !